package.json

```diff
@@ -9,14 +9,14 @@
   "scripts": {
     "start": "ember server",
     "build": "ember build",
-    "test": "ember test"
+    "test": "ember try:each"
```

Comment:
If I'm understanding this correctly, this allows us to test this component against multiple versions of Ember. Is that so we can get a heads up that things are going to break, and we can take appropriate action before it ruins your weekend?